<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icons/icons8-music-player-60.png</normaloff>:/img/icons/icons8-music-player-60.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="mute">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/icons/icons8-выключить-звук-50.png</normaloff>:/img/icons/icons8-выключить-звук-50.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="timeLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd"/>
   <addaction name="actionback"/>
   <addaction name="actionpause"/>
   <addaction name="actionplay"/>
   <addaction name="actionstop"/>
   <addaction name="actionnext"/>
   <addaction name="actionrepeat"/>
   <addaction name="actionrandom"/>
  </widget>
  <action name="actionadd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-добавить-песню-60.png</normaloff>:/img/icons/icons8-добавить-песню-60.png</iconset>
   </property>
   <property name="text">
    <string>add</string>
   </property>
   <property name="toolTip">
    <string>add file</string>
   </property>
  </action>
  <action name="actionback">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-перейти-в-начало-30.png</normaloff>:/img/icons/icons8-перейти-в-начало-30.png</iconset>
   </property>
   <property name="text">
    <string>back</string>
   </property>
   <property name="toolTip">
    <string>back</string>
   </property>
  </action>
  <action name="actionpause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-пауза-30.png</normaloff>:/img/icons/icons8-пауза-30.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="toolTip">
    <string>pause</string>
   </property>
  </action>
  <action name="actionplay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-воспроизведение-48.png</normaloff>:/img/icons/icons8-воспроизведение-48.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="toolTip">
    <string>play</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-стоп-96.png</normaloff>:/img/icons/icons8-стоп-96.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
   <property name="toolTip">
    <string>stop</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-конец-30.png</normaloff>:/img/icons/icons8-конец-30.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
   <property name="toolTip">
    <string>next</string>
   </property>
  </action>
  <action name="actionrepeat">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-retweet-48.png</normaloff>:/img/icons/icons8-retweet-48.png</iconset>
   </property>
   <property name="text">
    <string>repeat</string>
   </property>
   <property name="toolTip">
    <string>repeat</string>
   </property>
  </action>
  <action name="actionrandom">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-перемешать-48.png</normaloff>:/img/icons/icons8-перемешать-48.png</iconset>
   </property>
   <property name="text">
    <string>random</string>
   </property>
   <property name="toolTip">
    <string>random</string>
   </property>
  </action>
  <action name="actionmute">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons/icons8-выключить-звук-50.png</normaloff>:/img/icons/icons8-выключить-звук-50.png</iconset>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
   <property name="toolTip">
    <string>mute</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
